### PR TITLE
feat(@clayui/empty-state): LPD-21741 Add new SVG empty, success and search state

### PR DESCRIPTION
**Pull request by ilzamcmed**
Hey @matuzalemsteles,
This PR is not ready, I couldn't make the success and search svg animates, only the empty state.
But you can review how I made the changes.
Thanks

### CODE REVIEW

File: packages/clay-css/src/scss/components/_empty-state.scss [hunk 1]
```diff
@@ -63,3 +63,242 @@
 		@include clay-css(map-get($c-empty-state-sm, c-empty-state-footer));
 	}
 }
+
+// Empty State Animation
+.c-empty-state-animation {
```
Comment (matuzalemsteles) [inline packages/clay-css/src/scss/components/_empty-state.scss:68]: @pat270 Could you take a look at this?

**Review by matuzalemsteles — APPROVED** [review]
LGTM! Can you update the test threshold?

**Comment by matuzalemsteles** [comment]
This is good to go but I'll wait for @pat270 to take a look at the CSS part and see if he thinks it makes sense to keep this there or we move to adding inline to the component.

File: packages/clay-css/src/scss/components/_empty-state.scss (hunk 1 above)
Comment (pat270) [reply packages/clay-css/src/scss/components/_empty-state.scss:68]: @ilzamcmed we can inline the the styles that don't provide animation in the svg and only include the animation styles in Clay CSS.

**Review by pat270 — CHANGES_REQUESTED** [review]

File: packages/clay-css/src/scss/components/_empty-state.scss (hunk 1 above)
Comment (ilzamcmed) [reply packages/clay-css/src/scss/components/_empty-state.scss:68]: Thanks @pat270.
Done!

File: packages/clay-css/src/scss/components/_empty-state.scss [hunk 2]
```diff
@@ -63,3 +63,132 @@
...
+	.empty-state-animation {
```
Comment (pat270) [inline packages/clay-css/src/scss/components/_empty-state.scss:69]: Can we rename this `empty-state-svg`

File: packages/clay-css/src/scss/components/_empty-state.scss [hunk 3]
```diff
@@ -63,3 +63,132 @@
...
+		@media (prefers-reduced-motion: no-preference) {
+			.ship {
+				transform-origin: bottom;
+				animation: 6s ease-in-out infinite ship;
+			}
```
Comment (pat270) [inline packages/clay-css/src/scss/components/_empty-state.scss:74]: We need to add new lines after each rule set. Idk why `yarn format` doesn't do this.

File: packages/clay-css/src/scss/components/_empty-state.scss [hunk 4]
```diff
@@ -63,3 +63,132 @@
...
+					opacity: 0;
+				}
+				81% {
+					transform: scale(1);
+					opacity: 0;
+				}
+			}
+		}
+		.c-prefers-reduced-motion .ship,
+		.c-prefers-reduced-motion .wave {
```
Comment (pat270) [inline packages/clay-css/src/scss/components/_empty-state.scss:100]: This selector should be outside of `.empty-state-animation`. It should look like:
```
.c-empty-state-animation {
  &.c-prefers-reduced-motion,
  .c-prefers-reduced-motion {
    .ship,
    .wave {}
  }
}
```

Comment (ilzamcmed) [reply packages/clay-css/src/scss/components/_empty-state.scss:100]: Hey @pat270, thanks for pointing that out! 
Indeed the reduced motion was not working on the SVGs and after this change, it is still not working (I can still see the animation).

Should we make it more specific? Ex.:
```css
.c-empty-state-animation {
  &.c-prefers-reduced-motion,
  .c-prefers-reduced-motion .empty-state-svg {
    .ship,
    .wave {}
  }
}
```

Comment (ilzamcmed) [reply packages/clay-css/src/scss/components/_empty-state.scss:100]: Tested my suggestion and didn't work either...

File: packages/clay-css/src/scss/components/_empty-state.scss [hunk 5]
```diff
@@ -63,3 +63,132 @@
...
 	}
 }
+
+// Empty State Animation
+.c-empty-state-animation {
+	.empty-state-animation {
+		@media (prefers-reduced-motion: no-preference) {
+			.ship {
+				transform-origin: bottom;
+				animation: 6s ease-in-out infinite ship;
```
Comment (pat270) [inline packages/clay-css/src/scss/components/_empty-state.scss:73]: We need to alphabetize all CSS properties.

File: packages/clay-css/src/scss/components/_empty-state.scss (hunk 4 above)
Comment (pat270) [reply packages/clay-css/src/scss/components/_empty-state.scss:100]:
```
	&.c-prefers-reduced-motion,
	.c-prefers-reduced-motion {
		.empty-state-animation {
			.ship,
			.wave {
				animation: none;
			}
		}
	}
```
is working for me. You can put it at the end of the Sass statement too.

File: packages/clay-css/src/scss/components/_empty-state.scss [hunk 6]
```diff
@@ -63,3 +63,171 @@
 		@include clay-css(map-get($c-empty-state-sm, c-empty-state-footer));
 	}
 }
+
+// Empty State Animation
```
Comment (pat270) [inline packages/clay-css/src/scss/components/_empty-state.scss:67]: Need new line after this comment

Comment (ilzamcmed) [reply packages/clay-css/src/scss/components/_empty-state.scss:67]: Thanks @pat270 !

File: packages/clay-css/src/scss/components/_empty-state.scss (hunk 5 above)
Comment (pat270) [reply packages/clay-css/src/scss/components/_empty-state.scss:102]: animation needs to be delcared before transform-origin.

Comment (ilzamcmed) [reply packages/clay-css/src/scss/components/_empty-state.scss:101]: Thanks, Pat! I had this fixed but did some rollback and forgot to sort it again.
I believe everything is ok now

**Review by pat270 — APPROVED** [review]
@ilzamcmed I made a minor tweak in the css. `.c-prefers-reduced-motion` should have been `.c-prefers-reduced-motion &`. I also added to Cadmin.

**Pull request merged**